rest-api/spring-rest/src/main/java/uk/gov/gchq/gaffer/rest/controller/GremlinController.java

```diff
@@ -309,7 +311,17 @@ private Tuple2<Object, JSONObject> runGremlinQuery(final String gremlinQuery) {
         // OpenTelemetry hooks
         Span span = OtelUtil.startSpan(
             this.getClass().getName(), "Gremlin Request: " + UUID.nameUUIDFromBytes(gremlinQuery.getBytes(StandardCharsets.UTF_8)));
-        span.setAttribute("gaffer.gremlin.query", gremlinQuery);
+        span.setAttribute(OtelUtil.GREMLIN_QUERY_ATTRIBUTE, gremlinQuery);
+
+        Optional<Object> userOptional = graph.variables().get(GafferPopGraphVariables.USER);
```

Comment:
You can probably use the `gafferPopGraph` variable to get the `GafferPopGraphVariables` which have a `getUser` method to help simplify this